Grizzlies-SpyDuh/Models/Service.cs

```diff
@@ -4,4 +4,5 @@ public class Service
 {
     public int Id { get; set; }
     public string Name { get; set; }
+    public double Price { get; set; }
```

Comment:
I think it does make sense to have Price on here even though it doesn't exist on the ServiceLookup table in the db. That way we can say New Service and attach the price as needed. Good thinking.